package.json

```diff
@@ -12,29 +12,35 @@
   },
   "scripts": {
     "build": "gulp",
-    "create-push-notifications-service-user": "node dist/tools/createPushNotificationsServiceUser.js",
-    "create_db": "sequelize db:create",
-    "create_missing_users": "node dist/tools/create_missing_users.js",
-    "delete_db": "sequelize db:drop",
-    "jenkins": "cross-env NODE_ENV=test yarn delete_db; cross-env NODE_ENV=test yarn create_db && cross-env NODE_ENV=test yarn migrate && cross-env NODE_ENV=test yarn test",
+    "create-push-notifications-service-user": "babel-node tools/createPushNotificationsServiceUser.js",
+    "jenkins": "cross-env NODE_ENV=test yarn mongo:delete-db && cross-env NODE_ENV=test yarn test",
     "lint": "esw -c .eslintrc *.js server config --color",
     "lint:fix": "yarn lint --fix",
     "lint:watch": "yarn lint --watch",
-    "migrate": "sequelize db:migrate",
-    "migrate:undo": "sequelize db:migrate:undo",
+    "mongo:create-missing-users": "babel-node tools/createMissingUsers.js",
+    "mongo:delete-db": "babel-node tools/deleteDb.js",
+    "mongo:fail-on-db-exists": "babel-node tools/failOnDbExists.js",
+    "mongo:migrate:up": "db-migrate --config migrate.config.json up",
+    "mongo:migrate:down": "db-migrate --config migrate.config.json down",
     "report-coverage": "coveralls < ./coverage/lcov.info",
     "serve": "node ./dist/index.js",
     "snyk-protect": "snyk protect",
+    "sql:create-missing-users": "babel-node sql/tools/create_missing_users.js",
+    "sql:create-db": "sequelize db:create",
+    "sql:delete-db": "sequelize db:drop",
+    "sql:migrate": "sequelize db:migrate",
+    "sql:migrate:undo": "sequelize db:migrate:undo",
+    "sql:update-uuid": "babel-node sql/tools/addUUID.js",
+    "sql:to-mongo": "babel-node sql/tools/toMongo.js",
```

Comment:
Please add instructions to the `README.md` regarding how to migrate your existing notifications Postgres DB to Mongo.